### ISSUE TITLE
Make estela-cli a pip package

### ISSUE THREAD

**Issue by raymond1242** [bitmakerla/estela#84]
To go further, it would be nice if estela-cli is installed by the pip package manager

**Comment by breno** [comment]
@raymond1242 if you could find some time to prioritize this one it would be great

**Comment by breno** [comment]
@raymond1242 pip package is already created, the name was estela rather than estela-cli so i got confused. Safe to close this one too then.

**Comment by mgonnav** [comment]
Correct, the pip package for the `estela-cli` can be found in PyPI at https://pypi.org/project/estela/
